bridges/TwitterBridge.php

```diff
@@ -189,6 +189,9 @@ public function collectData(){
 			$item['fullname'] = htmlspecialchars_decode($tweet->getAttribute('data-name'), ENT_QUOTES);
 			// get author
 			$item['author'] = $item['fullname'] . ' (@' . $item['username'] . ')';
+			if($tweet->getAttribute('data-screen-name') !== $this->getInput('u')) {
```

Comment:
The provided name might not match exactly due to casing. Here is a slightly improved version:

```suggestion
			if(strcasecmp($tweet->getAttribute('data-screen-name'), $this->getInput('u'))) {
```

Reply:
This looks good, however this is not the only place where retweets are tested.

I will make a commit to fix both instances.

Reply:
Good point, thanks for taking care of that :+1: